Modules/Migration.lua

```diff
@@ -118,6 +118,22 @@ local migrationFunctions = {
                 Questie.db.global.isleOfQuelDanasPhase = 1
             end
         end
+    end,
+    [17] = function()
+        -- Preserve previous dungeon hide preference for both new flags
+        local previousHideInDungeons = Questie.db.profile.hideTrackerInDungeons
+
+        Questie.db.profile.minimizeTrackerInCombat = false
+        Questie.db.profile.minimizeTrackerInDungeons = previousHideInDungeons
+        Questie.db.profile.hideTrackerInCombat = false
+        Questie.db.profile.hideTrackerInDungeons = previousHideInDungeons
+
+        if Questie.db.profile.minimizeTrackerInDungeons == nil then
+            Questie.db.profile.minimizeTrackerInDungeons = true
+        end
+        if Questie.db.profile.hideTrackerInDungeons == nil then
+            Questie.db.profile.hideTrackerInDungeons = false
+        end
```

Comment:
I don't think we need this, do we? `Questie.db.profile.hideTrackerInDungeons` can not be `nil` as it has a default value and is around for quite some time 🤔